spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -34,7 +34,7 @@
       let(:saml_authn_request) { sp1_authnrequest }
 
       before do
-        subject.session[:saml_request_url] = saml_authn_request
+        subject.session[:user_return_to] = saml_authn_request
```

Comment:
This is revealing that we have an incorrect test here. This test is making an LOA1 request, but this is an IdV test, which only pertains to LOA3. In addition, the expectation is that the user is sent to the completions page, not directly to the SP.

We should not be setting `session[:user_return_to]` since we don't do that in the app.